src/types/IDPEntityDescriptor.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Do not edit this file it is auto-generated by italia-utils / gen-api-models.
+ * See https://github.com/teamdigitale/italia-utils
+ */
+/* tslint:disable */
+
+import * as t from "io-ts";
+
+// required attributes
+const IDPEntityDescriptorR = t.interface({
+  cert: t.readonlyArray(t.string, "array of string"),
+
+  entityID: t.string,
+
+  entryPoint: t.string,
+
+  logoutUrl: t.string
+});
+
+// optional attributes
+const IDPEntityDescriptorO = t.partial({});
```

Comment:
you can skip `IDPEntityDescriptorO` and the intersection below, this structure is used by `gen-api-models` for convenience, in your case you can use the single `t.interface` for simplicity - you can call it `IDPEntityDescriptor` instead of `IDPEntityDescriptorR`

Reply:
fixed on commit baa27f1